run_sdk_container

```diff
@@ -146,4 +147,4 @@ fi
 # Workaround: The SDK expects to be able to write to /etc/hosts
 $docker exec "$name" sh -c 'cp /etc/hosts /etc/hosts2; umount /etc/hosts ; mv /etc/hosts2 /etc/hosts'
 
-$docker exec $tty -i "$name" /home/sdk/sdk_entry.sh "$@"
+$docker exec $tty -i "$name" /mnt/host/source/src/scripts/sdk_lib/sdk_entry.sh "$@"
```

Comment:
This change is unrelated to allowing UID 500, is it? Maybe make a separate commit for it with some explanation. I suppose this is useful for testing changes in sdk_entry.sh without recreating SDK container.

Reply:
My idea was to fix it this way without having to create new SDK containers first. I can split it up.

Reply:
Sounds good. Please make it a separate commit, no need for a separate PR.